mlir/lib/Dialect/Tensor/IR/TensorOps.cpp

```diff
@@ -4439,7 +4439,7 @@ LogicalResult UnPackOp::canonicalize(UnPackOp unPackOp,
                                      PatternRewriter &rewriter) {
   /// pack(unpack(x)) -> x
```

Comment:
@MaheshRavishankar Hi, I am also curious about the notation here. The pattern here matches
```
%packed = tensor.pack %x .....
%unpacked = tensor.unpack %packed .....
```
Shall it be denoted as `unpack(pack(x))` instead of `pack(unpack(x))`?

Reply:
This is `unpack(pack(x))` because the unpack is operating on the result of the pack.

Reply:
Ah, yeah the comment does not match what the code is doing, good catch.

Reply:
I also fixed comment notation issue.